subsys/net/l2/canbus/6locan.c

```diff
@@ -19,7 +19,6 @@ LOG_MODULE_REGISTER(net_l2_canbus, CONFIG_NET_L2_CANBUS_LOG_LEVEL);
 #include <sys/byteorder.h>
 #include <net/ethernet.h>
 #include <net/net_ip.h>
-#include <string.h>
```

Comment:
We are using memmove.

Reply:
I think the point is that <string.h> is included twice on lines 19 & 22.

Reply:
Oh, I didn't unfold.